app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Idv
+  module Socure
+    class DocumentCaptureController < ApplicationController
+      include Idv::AvailabilityConcern
+      include IdvStepConcern
+      include DocumentCaptureConcern
+
+      before_action :confirm_not_rate_limited
+      before_action :confirm_step_allowed
+
+      def show
+        Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+          call('socure_document_capture', :view, true)
+
+        # document request
+        document_request = DocAuth::Socure::Requests::DocumentRequest.new(
+          document_capture_session_uuid: document_capture_session_uuid,
+          redirect_url: idv_socure_document_capture_url,
+          language: I18n.locale,
```

Comment:
the Socure Document Request documentation specifies simplified chinese as `zh-cn` ... providing our code `zh` is unexpected per the spec and renders the CaptureApp in English

Reply:
We make the zh-cn check here https://github.com/18F/identity-idp/pull/11289/files#diff-73f084413a18a98a63b45ea14072da5152ecb6aa9565151c79862057adf90e94R23-R26